### DISCUSSION TITLE
希望大纲能识别无序列表里的标题

### DISCUSSION THREAD

**Discussion by Amoscey** [tamlok/vnote#2130] [Ideas]
我和其他大纲软件一起用的……
软件里都是`- ## 标题`的形式，但是在vnote里面就看不见标题了
希望这种格式也能被识别。

**Comment by tamlok** [comment]
这不是基本md语法吧？

**Reply by Amoscey** [reply]
好吧，只能先忽略了

**Comment by Amoscey** [comment]
我突然发现如果是从其他应用打开跳转到vnote的话，会在预览模式显示这种大纲格式的标题`- ## 标题`，也能点击标题跳转。
但是编辑的时候又消失了……希望编辑的时候也能显示
